charts/fluid/fluid/values.yaml

```diff
@@ -6,13 +6,13 @@ workdir: /tmp
 
 dataset:
   controller:
-    image: registry.cn-hangzhou.aliyuncs.com/fluid/dataset-controller:v0.6.0-629adef
+    image: registry.cn-hangzhou.aliyuncs.com/fluid/dataset-controller:v0.6.0-b2f217d
```

Comment:
Not consistent with `DATASET_CONTROLLER_IMG` in Makefile?

Reply:
Updated